### ISSUE TITLE
DID from community-registry not being recognized

### ISSUE THREAD

**Issue by dmitrizagidulin** [digitalcredentials/learner-credential-wallet#254]
Something is off with the remote registry code -- a VC with a DID that's in community-registry is showing as not appearing in the registry by LCW detail screen.
To reproduce:

1. Load up latest `main` build, or Test Flight build 39.
2. Click on deep link: dccrequest://request?issuer=issuer.example.com&vc_request_url=https://verify.dcconsortium.org/request/credential&challenge=ke12345678-0001&auth_type=bearer
3. Accept that VC. The VC should have an issuer DID of `did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn`
4. This DID appears in Community Registry https://digitalcredentials.github.io/community-registry/registry.json

Expected behavior: This VC should have green checkmark, DID should appear in registry.
Actual behavior: Wallet says it's not found in registry.

**Comment by sethduffin** [comment]
@dmitrizagidulin I'll take a look at this.

**Comment by bmuramatsu** [comment]
@sethduffin have you addressed this?

**Comment by sethduffin** [comment]
@bmuramatsu I believe this was follow up [conversation in slack](https://atomicjolt.slack.com/archives/C0298868CR1/p1667576735029569) about the issue:

Nov 4, 2022 - @dmitrizagidulin 
> ah, wait, so I think I know what might be going on (with the community-registry).
so, one of the validation rules (that's not clearly spelled out, but it's there), is that the verificationMethod (key id) in the 'proof' has to match issuer.id (specifically, has to be authorized in the issuer's DID Document).

> but in that test VC from the deep link, those two are mis-matched.

> so what is likely happening, is that the issuer.id is indeed in the community-registry. But the validation is throwing a mismatch error, and it's showing up as 'issuer not recognized'

Happy to work on this further, just let me know!

**Comment by bmuramatsu** [comment]
Ok, Dmitri please close if appropriate.

**Comment by dmitrizagidulin** [comment]
@bmuramatsu right, ok, so, I think I've been keeping this issue open as a reminder to open a corresponding issue on our VC library. So I might as well just move this issue to that lib.
